runtime/doc/vim_diff.txt

```diff
@@ -134,49 +145,34 @@ Options:
   'tabline' %@Func@foo%X can call any function on mouse-click
   'winhighlight' window-local highlights
 
-Variables:
-  |v:event|
-  |v:exiting|
-  |v:progpath| is always absolute ("full")
-  |v:windowid| is always available (for use by external UIs)
-
-Commands:
-  |:checkhealth|
-  |:drop| is available on all platforms
-  |:Man| is available by default, with many improvements such as completion
-  |:tchdir| tab-local |current-directory|
-
-Functions:
-  |dictwatcheradd()| notifies a callback whenever a |Dict| is modified
-  |dictwatcherdel()|
-  |menu_get()|
-  |msgpackdump()|, |msgpackparse()| provide msgpack de/serialization
-
 Events:
   |DirChanged|
   |TabNewEntered|
   |TermClose|
   |TermOpen|
   |TextYankPost|
```

Comment:
`TextYankPost` and `v:event` are present in Vim now.

Reply:
`v:event` is still relevant, this list is intended to point the user to things that are different, not exhaustively list the precise intersection. `v:event` in Nvim still covers many more events (and will continue to expand) vs Vim.